docs/11-dev-guide.md

```diff
@@ -11,7 +11,10 @@ copy that you built yourself.
 
 ### Install Prerequisite Software
 
-You will need the following software:
+Information about the prerequesite software and how to get an environment with these prerequesites already installed can
+be found in the [deployment guide](02-deployment-guide.md#install-prerequisite-software)
+
+When running commands inside your EC2 environment, you will need the following software:
```

Comment:
This seems a bit confusing. This guide is about a development environment, rather than specifically inside an EC2.